docs/content/docs/try-flink/local_installation.md

```diff
@@ -24,71 +24,139 @@ specific language governing permissions and limitations
+
+To quickly stop the cluster and all running components, you can use the provided script:
+
+```bash
+$ ./bin/stop-cluster.sh
+```
+
+## Submitting a Flink job
 
-Releases of Flink come with a number of example Jobs.
-You can quickly deploy one of these applications to the running cluster. 
+Flink provides a CLI tool, __bin/flink__, that can run programs packaged as Java ARchives (JAR)
+and control their execution. Submitting a [job](https://ci.apache.org/projects/ﬂink/ﬂink-docs-stable/concepts/glossary.html#ﬂink-job) means uploading the job’s JAR ﬁle and related dependencies to the running Flink cluster
+and executing it.
+
+Flink releases come with example jobs, which you can ﬁnd in the __examples/__ folder.
+
+To deploy the example word count job to the running cluster, issue the following command:
 
 ```bash
 $ ./bin/flink run examples/streaming/WordCount.jar
-$ tail log/flink-*-taskexecutor-*.out
-  (to,1)
-  (be,1)
-  (or,1)
-  (not,1)
-  (to,2)
-  (be,2)
 ```
 
-Additionally, you can check Flink's [Web UI](http://localhost:8081) to monitor the status of the Cluster and running Job.
+You can verify the output by viewing the logs:
 
-## Step 4: Stop the Cluster
+```bash
+$ tail log/flink-*-taskexecutor-*.out
+```
 
-When you are finished you can quickly stop the cluster and all running components.
+Sample output:
 
 ```bash
-$ ./bin/stop-cluster.sh
+  (nymph,1)
+  (in,3)
+  (thy,1)
+  (orisons,1)
+  (be,4)
+  (all,2)
+  (my,1)
+  (sins,1)
+  (remember,1)
+  (d,4)
 ```
+
+Additionally, you can check Flink's [web UI](http://localhost:8081) to monitor the status of the cluster and running job.
+
+You can view the data flow plan for the execution:
+
+![data flow plan](dataflowplan.png)
+
+Here for the job execution, Flink has two operators. The ﬁrst is the source operator which reads data from the
+collection source. The second operator is the transformation operator which aggregates counts of words. Learn
+more about [DataStream operators](https://ci.apache.org/projects/ﬂink/ﬂink-docs-stable/dev/stream/operators/).
+
+You can also look at the timeline of the job execution:
+
+![data flow timeline](timeline.png)
+
+You have successfully ran a [Flink application](https://ci.apache.org/projects/ﬂink/ﬂink-docs-stable/concepts/glossary.html#ﬂink-application)! Feel free to select any other JAR archive from the __examples/__
+folder or deploy your own job!
+
+# Summary
+
+In this guide, you downloaded Flink, explored the project directory, started and stopped a local cluster, and
+submitted a sample Flink job!
+
+To learn more about Flink fundamentals, check out the [concepts](https://ci.apache.org/projects/ﬂink/ﬂink-docs-release-1.12/concepts/index.html) section. If you want to try something more hands-on, try one of the tutorials.
+
+_________________________________
+[Want to contribute translation?](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications)
```

Comment:
What do you think about putting this in the footer template so it's on every page automatically? 

See `docs/layouts/partials/docs/footer.html`